backend/src/routing/pendingCourses.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import { Params } from '../enums/routesEnum';
+import withAuth from '../middlewares/authMiddleware';
+import { USER_ROLES } from '../config/constants';
+import getPendingCourses from '../controllers/pendingCourses/getPendingCourses';
```

Comment:
All this stuff should be imported using absolute paths! (you're importing components from other src-level dir)